include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -46,8 +48,11 @@ namespace ens {
  * ensmallen website.
  *
  * @tparam SelectionPolicy The selection strategy used for the evaluation step.
+ * @tparam transformationPolicy The transformation strategy used to 
+ *       map cooridnates to the desired domain.
```

Comment:
```suggestion
 * @tparam TransformationPolicy The transformation strategy used to 
 *       map coordinates to the desired domain.
```

Do you think you can provide a little more documentation on what is expected of a `TransformationPolicyType`?

Reply:
Alright!